### ISSUE TITLE
Glowing review! <3

### ISSUE THREAD

**Issue by Selloutsmakemesade** [Nexus-Mods/Nexus-Mod-Manager#1075]
tl;dr at the bottom lol.

What is this... I'm totally down to support a site, 3euro is totally reasonable and you honestly could have just said "yo, fast download 3euro!" and I'd have been like, sign me up. But to make this entire process require 3 website domains, a 12 character password because "what the fück????", and then to hold payment for... again why... 

Is this the mods from hipixel skyblocks server in minecraft by chance xD 

~main take away~ (tl;dr under this) Leggit. easy, simple, transparent. It's like this is some suit running the worlds weirdest hedgefund to make money on the side. Run by some drugged up "programmer" whose experience comes from the incredible machine xD 

tl;dr ~ Don't give people waiting on downloads ample time to flame you. It doesn't matter how many accounts I have to pretend to make when you make it this stupid to do something that should be easy xD

**Comment by DuskDweller** [comment]
Wrong section, this software is now community maintained. You need to address your rants toward the website support.
